app/services/supported_ssg_updater.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# This class updates supported SSGs yaml
+class SupportedSsgUpdater
+  require 'yaml'
+
+  KEYS_TO_REMOVE = %w[upstream_version brew_url].freeze
```

Comment:
You mention 2 keys here, but the tests only cover one.

Reply:
the tests are checking for the absence of both at the same time

Reply:
No? 
![Screenshot from 2022-02-24 09-53-23](https://user-images.githubusercontent.com/649130/155491097-9da5008a-b71f-4025-af99-703a63ea6101.png)

Reply:
What I tried to say is that the `upstream_version` is not even in the fixture yaml :disappointed: